src/libsyntax/ext/tt/macro_parser.rs

```diff
@@ -161,7 +162,7 @@ pub fn count_names(ms: &[TokenTree]) -> usize {
     })
 }
 
-pub fn initial_matcher_pos(ms: Rc<Vec<TokenTree>>, sep: Option<Token>, lo: BytePos)
+pub fn initial_matcher_pos(ms: Vec<TokenTree>, sep: Option<Token>, lo: BytePos)
```

Comment:
I believe this means that we are now copying/moving the Vec; perhaps we only need `&[TokenTree]` here?